content/getting-started/accounts.md

```diff
@@ -0,0 +1,15 @@
+---
+menu:
+  main:
+    parent: getting-started
+title: Accounts
+weight: -100
+---
+
+## Setting up your account
+
+You will need a Cloud Foundry account before continuing. If you don't already have an account, please [create an issue in the DevOps issue tracker](https://github.com/18F/DevOps/issues/new) and assign it to [@dlapiduz](https://github.com/dlapiduz) or [@ozzyjohnson](https://github.com/ozzyjohnson).
```

Comment:
@dlapiduz @ozzyjohnson Is [the page around creating users](https://docs.cloud.gov/ops/create-user/) still up-to-date, now that the platform allows us to [invite users](https://login.cloud.gov/invitations/new)?